app/views/admin/members/show.html.haml

```diff
@@ -41,9 +41,9 @@
               %br
               #{@member.postal_code} #{@member.city}
             %p
-              #{@member.phone_number}
+              #{TelephoneNumber.parse(@member.phone_number).international_number()}
```

Comment:
Style nitpick: When calling a function with no arguments, you can leave out the `()` at the end (like properties in Python).